.github/workflows/pylint.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         pip install pylint
     - name: Add dependencies about external libraries
       run: |
+        pip install h5py
```

Comment:
Please add this to the unit test action as well.

Reply:
I planned this later becuase the unit test wasn't updated in this PR.
But you metioned it, so I will apply it.

Reply:
It is not a problem that you postpone testing, but you should let reviewers know!

Reply:
+ Even if you planned it as TODO, I think it is necessary to make the unit test not failing.

Reply:
Okay! Then, I will add it in this PR.

Reply:
Hmm, the unit test is still failing, so I think you should inspect it here.
If it is the problem of the test, you can resolve it in another PR later.
If the test _was_ correct but it now fails because of this PR, I think you should resolve it here because it could "blind" us from other test failures (we might not see the other test failures since we know and expect that the test should fail).

Reply:
As we discussed offline, the error has been occurred even before this PR, so it will be resolved in another PR.